packages/admin/src/pages/super-admin-dashboard/user/[id]/change-roles.page.tsx

```diff
@@ -18,13 +18,23 @@ type PageBodyResponse = {
 
 export async function getServerSideProps(context: GetServerSidePropsContext) {
   const userId = context.params?.id as string;
+  const findAndApplicantRoles = ['1', '2'];
```

Comment:
May as well keep the all caps naming convention for these consts 

```suggestion
  const APPLICANT_ROLE_IDS = ['1', '2'];
```